### ISSUE TITLE
[Bug]: 'ArgumentHelper' has no attribute 'enable_prefix_caching'

### ISSUE THREAD

**Issue by xiaohangguo** [InternLM/lmdeploy#1592]
### Checklist

- [X] 1. I have searched related issues but cannot get the expected help.
- [X] 2. The bug has not been fixed in the latest version.

### 🐛 Describe the bug

注释掉 `286`行 
`  ArgumentHelper.enable_prefix_caching(pt_group)`
以后会有这个问题：

如果不注释的话，会有这个没有这个属性的问题
```
(xtuner) [lvshuhang@node62 benchmark]$ python3 profile_throughput.py ./ShareGPT_V3_unfiltered_cleaned_split.json  /public/home/lvshuhang/model_space/workspace/internlm2-chat-7b --tp 1
Traceback (most recent call last):
  File "/public/home/lvshuhang/lmdeploy/benchmark/profile_throughput.py", line 343, in <module>
    main()
  File "/public/home/lvshuhang/lmdeploy/benchmark/profile_throughput.py", line 304, in main
    args = parse_args()
  File "/public/home/lvshuhang/lmdeploy/benchmark/profile_throughput.py", line 286, in parse_args
    ArgumentHelper.enable_prefix_caching(pt_group)
AttributeError: type object 'ArgumentHelper' has no attribute 'enable_prefix_caching'
```

```

Traceback (most recent call last):
  File "/public/home/lvshuhang/lmdeploy/benchmark/profile_throughput.py", line 343, in <module>
    main()
  File "/public/home/lvshuhang/lmdeploy/benchmark/profile_throughput.py", line 329, in main
    engine = Engine(args.model_path, engine_config, csv=args.csv)
  File "/public/home/lvshuhang/lmdeploy/benchmark/profile_throughput.py", line 73, in __init__
    tm_model = TurboMind.from_pretrained(model_path,
  File "/public/home/lvshuhang/miniconda3/envs/xtuner/lib/python3.9/site-packages/lmdeploy/turbomind/turbomind.py", line 396, in from_pretrained
    return cls(model_path=pretrained_model_name_or_path,
  File "/public/home/lvshuhang/miniconda3/envs/xtuner/lib/python3.9/site-packages/lmdeploy/turbomind/turbomind.py", line 170, in __init__
    self.model_comm = self._from_hf(model_source=model_source,
  File "/public/home/lvshuhang/miniconda3/envs/xtuner/lib/python3.9/site-packages/lmdeploy/turbomind/turbomind.py", line 292, in _from_hf
    model_comm = _tm.AbstractTransformerModel.create_llama_model(
RuntimeError: [TM][ERROR] CUDA runtime error: CUDA driver version is insufficient for CUDA runtime version /lmdeploy/src/turbomind/utils/cuda_utils.h:326 

### Reproduction

相关版本和命令如下：


`python3 profile_throughput.py ./ShareGPT_V3_unfiltered_cleaned_split.json  /public/home/lvshuhang/model_space/workspace/internlm2-chat-7b `

(xtuner) [lvshuhang@node62 benchmark]$ nvcc --version
nvcc: NVIDIA (R) Cuda compiler driver
Copyright (c) 2005-2022 NVIDIA Corporation
Built on Wed_Sep_21_10:33:58_PDT_2022
Cuda compilation tools, release 11.8, V11.8.89
Build cuda_11.8.r11.8/compiler.31833905_0

```
(xtuner) [lvshuhang@node62 benchmark]$ nvidia-smi
Tue May 14 21:51:08 2024       
+-----------------------------------------------------------------------------+
| NVIDIA-SMI 520.61.05    Driver Version: 520.61.05    CUDA Version: 11.8     |
|-------------------------------+----------------------+----------------------+
| GPU  Name        Persistence-M| Bus-Id        Disp.A | Volatile Uncorr. ECC |
| Fan  Temp  Perf  Pwr:Usage/Cap|         Memory-Usage | GPU-Util  Compute M. |
|                               |                      |               MIG M. |
|===============================+======================+======================|
|   0  NVIDIA A100 80G...  On   | 00000000:4F:00.0 Off |                    0 |
| N/A   25C    P0    41W / 300W |      2MiB / 81920MiB |      0%      Default |
|                               |                      |             Disabled |
+-------------------------------+----------------------+----------------------+
|   1  NVIDIA A100 80G...  On   | 00000000:50:00.0 Off |                    0 |
| N/A   26C    P0    43W / 300W |      2MiB / 81920MiB |      0%      Default |
|                               |                      |             Disabled |
+-------------------------------+----------------------+----------------------+
|   2  NVIDIA A100 80G...  On   | 00000000:A0:00.0 Off |                    0 |
| N/A   24C    P0    43W / 300W |      2MiB / 81920MiB |      0%      Default |
|                               |                      |             Disabled |
+-------------------------------+----------------------+----------------------+
|   3  NVIDIA A100 80G...  On   | 00000000:A4:00.0 Off |                    0 |
| N/A   25C    P0    40W / 300W |      2MiB / 81920MiB |      0%      Default |
|                               |                      |             Disabled |
+-------------------------------+----------------------+----------------------+
                                                                               
+-----------------------------------------------------------------------------+
| Processes:                                                                  |
|  GPU   GI   CI        PID   Type   Process name                  GPU Memory |
|        ID   ID                                                   Usage      |
|=============================================================================|
|  No running processes found                                                 |
+-----------------------------------------------------------------------------+
```

```
(xtuner) [lvshuhang@node62 benchmark]$ python
Python 3.9.0 (default, Nov 15 2020, 14:28:56) 
[GCC 7.3.0] :: Anaconda, Inc. on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import torch
torch>>> torch.cuda.is_available()
True
```


```
(xtuner) [lvshuhang@node62 benchmark]$ pip list | grep "lmdeploy"
lmdeploy                                 0.4.1
```
```
(xtuner) [lvshuhang@node62 benchmark]$ pip list | grep "torch"
torch                                    2.1.0+cu118
torchaudio                               2.1.0+cu118
torchvision                              0.16.0+cu118
```

### Environment


(xtuner) [lvshuhang@node62 benchmark]$ lmdeploy check_env
```
sys.platform: linux
Python: 3.9.0 (default, Nov 15 2020, 14:28:56) [GCC 7.3.0]
CUDA available: True
MUSA available: False
numpy_random_seed: 2147483648
GPU 0,1,2,3: NVIDIA A100 80GB PCIe
CUDA_HOME: /usr/local/cuda-11.8
NVCC: Cuda compilation tools, release 11.8, V11.8.89
GCC: gcc (GCC) 7.3.1 20180303 (Red Hat 7.3.1-5)
PyTorch: 2.1.0+cu118
PyTorch compiling details: PyTorch built with:
  - GCC 9.3
  - C++ Version: 201703
  - Intel(R) oneAPI Math Kernel Library Version 2022.2-Product Build 20220804 for Intel(R) 64 architecture applications
  - Intel(R) MKL-DNN v3.1.1 (Git Hash 64f6bcbcbab628e96f33a62c3e975f8535a7bde4)
  - OpenMP 201511 (a.k.a. OpenMP 4.5)
  - LAPACK is enabled (usually provided by MKL)
  - NNPACK is enabled
  - CPU capability usage: AVX512
  - CUDA Runtime 11.8
  - NVCC architecture flags: -gencode;arch=compute_50,code=sm_50;-gencode;arch=compute_60,code=sm_60;-gencode;arch=compute_70,code=sm_70;-gencode;arch=compute_75,code=sm_75;-gencode;arch=compute_80,code=sm_80;-gencode;arch=compute_86,code=sm_86;-gencode;arch=compute_37,code=sm_37;-gencode;arch=compute_90,code=sm_90
  - CuDNN 8.7
  - Magma 2.6.1
  - Build settings: BLAS_INFO=mkl, BUILD_TYPE=Release, CUDA_VERSION=11.8, CUDNN_VERSION=8.7.0, CXX_COMPILER=/opt/rh/devtoolset-9/root/usr/bin/c++, CXX_FLAGS= -D_GLIBCXX_USE_CXX11_ABI=0 -fabi-version=11 -fvisibility-inlines-hidden -DUSE_PTHREADPOOL -DNDEBUG -DUSE_KINETO -DLIBKINETO_NOROCTRACER -DUSE_FBGEMM -DUSE_QNNPACK -DUSE_PYTORCH_QNNPACK -DUSE_XNNPACK -DSYMBOLICATE_MOBILE_DEBUG_HANDLE -O2 -fPIC -Wall -Wextra -Werror=return-type -Werror=non-virtual-dtor -Werror=bool-operation -Wnarrowing -Wno-missing-field-initializers -Wno-type-limits -Wno-array-bounds -Wno-unknown-pragmas -Wno-unused-parameter -Wno-unused-function -Wno-unused-result -Wno-strict-overflow -Wno-strict-aliasing -Wno-stringop-overflow -Wno-psabi -Wno-error=pedantic -Wno-error=old-style-cast -Wno-invalid-partial-specialization -Wno-unused-private-field -Wno-aligned-allocation-unavailable -Wno-missing-braces -fdiagnostics-color=always -faligned-new -Wno-unused-but-set-variable -Wno-maybe-uninitialized -fno-math-errno -fno-trapping-math -Werror=format -Werror=cast-function-type -Wno-stringop-overflow, LAPACK_INFO=mkl, PERF_WITH_AVX=1, PERF_WITH_AVX2=1, PERF_WITH_AVX512=1, TORCH_DISABLE_GPU_ASSERTS=ON, TORCH_VERSION=2.1.0, USE_CUDA=ON, USE_CUDNN=ON, USE_EXCEPTION_PTR=1, USE_GFLAGS=OFF, USE_GLOG=OFF, USE_MKL=ON, USE_MKLDNN=ON, USE_MPI=OFF, USE_NCCL=1, USE_NNPACK=ON, USE_OPENMP=ON, USE_ROCM=OFF, 

TorchVision: 0.16.0+cu118
LMDeploy: 0.4.1+ca4de27
transformers: 4.36.2
gradio: 3.50.2
fastapi: 0.104.1
pydantic: 2.6.4
triton: 2.1.0
```
```

### Error traceback

```Shell
1
```

**Comment by aaronlyt** [comment]
had been solved? version-0.4.1 still have this problem

**Comment by lvhan028** [comment]
Please use the benchmark scripts in v0.4.1, not in the latest main branch.
Because the latest main involves prefix caching option, which is not in v0.4.1
